src/lib/y2firewall/clients/auto.rb

```diff
@@ -215,9 +270,20 @@ def check_profile_for_errors
 
       # Depending on the profile it activates or deactivates the firewalld
       # service
+      #
+      # Note: it always activates the service on the target
       def activate_service
-        enable? ? firewalld.enable! : firewalld.disable!
```

Comment:
Dos not work by some reason?

Reply:
I don't remmember. I think there was a problem but I'm not sure now. Will retest.